packages/shared/src/components/squads/layout/SquadDirectoryNavbar.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { ComponentProps, ReactElement } from 'react';
+import classNames from 'classnames';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonVariant,
+} from '../../buttons/Button';
+import { useHorizontalScrollHeader } from '../../HorizontalScroll/useHorizontalScrollHeader';
+import { ArrowIcon } from '../../icons';
+import { useViewSize, ViewSize } from '../../../hooks';
+
+interface SquadNavbarItemProps extends Omit<ComponentProps<'li'>, 'onClick'> {
+  buttonSize: ButtonSize;
+  isActive: boolean;
+  label: string;
+  onClick: ButtonProps<'a'>['onClick'];
+  path?: string;
+}
+
+export const SquadDirectoryNavbarItem = ({
+  buttonSize,
+  className,
+  isActive,
+  label,
+  onClick,
+  path,
+  ...attrs
+}: SquadNavbarItemProps): ReactElement => (
+  <li
+    {...attrs}
+    className={classNames(
+      'relative py-3',
+      'after:absolute after:bottom-0 after:left-0 after:right-0 after:mx-auto after:w-14 after:border-b-2',
```

Comment:
What are you adding here, like a css arrow?

Reply:
This is the border at bottom of the active element, this line has a fixed width so i made it with those classes on a `::after`

Reply:
Ah shouldn't it just be w-full of each element then?